qcodes/utils/__init__.py

```diff
@@ -20,20 +20,23 @@
 from .numpy_utils import list_of_data_to_maybe_ragged_nd_array
 from .partial_utils import partial_with_docstring
 from .path_helpers import get_qcodes_path, get_qcodes_user_path
+from .snapshot_helpers import ParameterDiff, diff_param_values, extract_param_values
 from .spyder_utils import add_to_spyder_UMR_excludelist
 from .threading_utils import RespondingThread, thread_map
 
 __all__ = [
     "DelayedKeyboardInterrupt",
     "DelegateAttributes",
     "NumpyJSONEncoder",
+    "ParameterDiff",
     "QCoDeSDeprecationWarning",
     "RespondingThread",
     "attribute_set_to",
     "checked_getattr",
     "convert_legacy_version_to_supported_version",
     "deep_update",
     "deprecate",
+    "diff_param_values",
```

Comment:
should this also be added?
```suggestion
    "diff_param_values",
    "extract_param_values",
```